### PR TITLE
adding in new content for partners

### PR DESCRIPTION
**Pull request by ksigler7**
@Whatthefoxsays - you can go ahead and start editing this, I just have two more things I need to do on it:
[ ] add the images properly into the source code
[ ] add links to the appropriate place in the UI in each step

### CODE REVIEW

File: source/Integrate/Partners/account-migration.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  description: Sometimes if a SendGrid partner no longer supports SendGrid core functionality, you may want to migrate your account to a regular SendGrid account.
+  keywords: migrate, partner sunset
+title: SendGrid account migration
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+
+If you are migrating from one SendGrid account to another, there are several things to keep in mind - make sure to stop sending mail until you migrate to a new, warmed up account, and make sure export data from your old account, and clean up the data described here so that you can import it into your new account.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: Maybe link to "how to warm up an ip address" here?

File: source/Integrate/Partners/account-migration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+If you have domain, link, and IP authentication set up, remove 5 CNAME or MX/TXT records and 1 A record from your domain’s DNS. To verify the records delete from your DNS, go to the [Sender Authentication page](https://app.sendgrid.com/settings/sender_auth) in the UI and check the DNS column:
+
+![]({{root_url}}/images/dns-records.png "Where to find your existing DNS records")
+
+To set up sender authentication in your new account, check out [Setting up domain authentication]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.html), [Setting up reverse DNS]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.html), and [Setting up link branding]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.html).
+
+2. **Migrate Unsubscribes** - To download your unsubscribes, on the [Global unsubscribes page](https://app.sendgrid.com/suppressions/global_unsubscribes), select the action cog, and then click *Export CSV*.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: That damn action cog....😝I checked the UI and it says "Download as CSV" on the Global unsubscribes page. (Which is, of course, different from what it says on the unsubscribe groups page, but what you gonna do?)

File: source/Integrate/Partners/account-migration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+To set up your unsubscribes in your new account, copy them into the UI by going to [Global](https://app.sendgrid.com/suppressions/global_unsubscribes) or [Group](https://app.sendgrid.com/suppressions/group_unsubscribes) Unsubscribes page, selecting the action cog and then adding them to the list. You can add them with the API by using the [Suppressions methods](https://sendgrid.api-docs.io/v3.0/suppressions-global-suppressions/add-recipient-addresses-to-the-global-suppression-group)
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: Action cog = action menu

File: source/Integrate/Partners/account-migration.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+3. **Migrate templates** - From the [Marketing Campaigns templates](https://sendgrid.com/marketing_campaigns/ui/marketing_templates) and the [Transactional templates](https://sendgrid.com/dynamic_templates) pages in the UI, export the HTML of any templates you want to migrate.
+
+To add your templates in the new account, navigate to either the [Marketing Campaigns templates](https://sendgrid.com/marketing_campaigns/ui/marketing_templates) or the [Transactional templates](https://sendgrid.com/dynamic_templates) page in the UI, create a new template, select to use the code editor, and paste the HTML of the exported template into the editor.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: would it be too much to link to the code editor page here? Link overload?

Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: To add your templates in the new account, navigate to either the [Marketing Campaigns templates](https://sendgrid.com/marketing_campaigns/ui/marketing_templates) or the [Transactional templates](https://sendgrid.com/dynamic_templates) page in the UI, click **create a new template**, and then select the code editor.  Then paste the HTML of the exported template into the editor.

File: source/Integrate/Partners/account-migration.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+4. **Migrate Marketing Campaigns Contacts** - From the [Marketing Campaigns contacts](https://sendgrid.com/marketing_campaigns/ui/contacts) page, click into your [Global All Contacts](https://sendgrid.com/marketing_campaigns/ui/all_contacts) list. Then export a CSV of your contacts.
+
+To upload your contacts into your new account, from the [Contacts page](https://sendgrid.com/marketing_campaigns/ui/contacts), click *Add contacts* and then *Upload CSV*.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: To upload your contacts into your new account, from the [Contacts page](https://sendgrid.com/marketing_campaigns/ui/contacts), click **Add contacts** and then **Upload CSV**.

Comment (Whatthefoxsays) [reply source/Integrate/Partners/account-migration.md]: I guess I should specify above that the UI items need an extra asterisk to be bold

File: source/Integrate/Partners/account-migration.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+3. **Migrate templates** - From the [Marketing Campaigns templates](https://sendgrid.com/marketing_campaigns/ui/marketing_templates) and the [Transactional templates](https://sendgrid.com/dynamic_templates) pages in the UI, export the HTML of any templates you want to migrate.
+
+To add your templates in the new account, navigate to either the [Marketing Campaigns templates](https://sendgrid.com/marketing_campaigns/ui/marketing_templates) or the [Transactional templates](https://sendgrid.com/dynamic_templates) page in the UI, create a new template, select to use the code editor, and paste the HTML of the exported template into the editor.
+
+4. **Migrate Marketing Campaigns Contacts** - From the [Marketing Campaigns contacts](https://sendgrid.com/marketing_campaigns/ui/contacts) page, click into your [Global All Contacts](https://sendgrid.com/marketing_campaigns/ui/all_contacts) list. Then export a CSV of your contacts.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: 4. **Migrate Marketing Campaigns Contacts** - From the [Marketing Campaigns contacts](https://sendgrid.com/marketing_campaigns/ui/contacts) page, click into your [All Contacts](https://sendgrid.com/marketing_campaigns/ui/all_contacts) list. Then export a CSV of your contacts by hovering over the action menu and clicking **Export**.

File: source/Integrate/Partners/account-migration.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+To upload your contacts into your new account, from the [Contacts page](https://sendgrid.com/marketing_campaigns/ui/contacts), click *Add contacts* and then *Upload CSV*.
+
+5. **Review existing API keys** - If you are using the SendGrid API, double check your accounts [list of API keys](https://app.sendgrid.com/settings/api_keys). API keys are not exportable, but it's good to make sure that you're aware of all of the existing API keys in your account so that you can create new ones to match in your new account and update your integrations with the new API keys.
+
+6. **Delete teammates** - Teammate names have to be unique across SendGrid, so if you want to continue to use your teammate name, delete it from your original account to add it to your new account.
+
+7. **Create new subusers** - If you are using subusers, navigate to the [subusers page](https://app.sendgrid.com/settings/subusers) in the UI and click *Export data* to download all of the information on your current subsuers.
```
Comment (Whatthefoxsays) [inline source/Integrate/Partners/account-migration.md]: Add more asterisks for bold UI items

Comment (Whatthefoxsays) [reply source/Integrate/Partners/account-migration.md]: Actually, just apply that to the whole page. haha

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**